resources/css/bem/header-nav-v4.less

```diff
@@ -3,88 +3,74 @@
 
 .header-nav-v4 {
   @_top: header-nav-v4;
-  @_gutter-x: 5px;
-  @_gutter-x-list: 10px;
-  @_gutter-y: 4px;
-  @_gutter-y-desktop: 15px;
+  --header-nav-item-padding-y: 4px;
+
   list-style: none;
   display: none;
   flex-wrap: wrap;
-  margin: 0 -@_gutter-x;
-  padding: (10px - @_gutter-y) 0;
+  gap: 10px;
+  margin: 0;
+  padding: 0;
   font-size: @font-size--normal;
   position: relative;
 
   @media @desktop {
+    --header-nav-item-padding-y: 15px;
     font-size: @font-size--title-small;
-    padding-top: 0;
-    padding-bottom: 0;
     display: flex;
   }
 
   &--breadcrumb {
     display: flex;
+
+    @media @mobile {
+      --header-nav-item-padding-y: 10px;
```

Comment:
this looks like should/can be in the main block instead. I don't know where the 4px version is even visible.